File: 2298-count-integers-with-even-digit-sum/solution.java
class Solution {
    public int countEven(int num) {
        int c=0;
        for(int i=1;i<=num;i++){
            int s=0;
            int t=i;
            while(t!=0){
                int m=t%10;
                s+=m;
                t/=10;
            }
            if(s%2==0){
                c++;
            }
        }
        
        return c;
    }
}

File: 2298-count-integers-with-even-digit-sum/solution.py
class Solution:
    def countEven(self, num: int) -> int:
        list1=[]
        for i in range(1,num+1):
            list1.append(str(i))
        c=0
        for i in list1:
            s=0
            for j in i:
                s=s+int(j)
            if s%2==0:
                c+=1
        return c
                





            
